templates/services.hbs

```diff
@@ -0,0 +1,15 @@
+{{!-- services --}}
+<section class="services">
+  <div class="services-container">
+    <div class="services-info">
+      <div class="services-eyebrow">{{eyebrow}}</div>
```

Comment:
Use tag span or p instead of div

Reply:
Thanks for reviewing.
Fixed issue: 07a96fcd737309fbdad7f6232aa66ad9450e6a3b